addons/website_sale_referral/models/res_config_settings.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from odoo.tools import float_repr
+from ast import literal_eval
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    reward_value = fields.Monetary(string="Reward Value", currency_field='company_currency_id', readonly=False)
+    company_currency_id = fields.Many2one(
```

Comment:
There is already a field on res.config model. See: https://github.com/odoo/odoo/blame/master/addons/account/models/res_config_settings.py#L11-L12

You could move the field to base in the master branch. By the way, your PR is supposed to target the 13.0 right ?

Reply:
As talked, will be done in fast forward